ch03/flannel/bootstrap.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+ETCD_URL=https://github.com/coreos/etcd/releases/download/v2.0.13/etcd-v2.0.13-linux-amd64.tar.gz
+FLANNEL_URL=https://github.com/coreos/flannel/releases/download/v0.5.1/flannel-0.5.1-linux-amd64.tar.gz
+
+# Install latest Docker
+curl -sSL https://get.docker.com/ubuntu/ | sudo sh
```

Comment:
I will merge it but the install link has changed to /get.docker.com otherwise you get in aufs issues on ubuntu 14.04. I need to correct all vagrantfiles.